index.html

```diff
@@ -41,16 +41,7 @@
             if(lang) {
                 redirect(lang);
             } else {
-                var host = window.location.host,
-                    socketUrl;
-                if((/www\.binary\.com/i).test(host)) {
-                    socketUrl = 'wss://ws.binaryws.com/websockets/v3';
-                } else if((/binaryqa/i).test(host)) {
-                    socketUrl = 'wss://' + host + '/websockets/v3';
-                } else {
-                    socketUrl = 'wss://www2.binary.com/websockets/v3';
-                }
-                var ws = new WebSocket(socketUrl);
+                var ws = new WebSocket('wss://ws.binaryws.com/websockets/v3');
```

Comment:
I tried to make this index.html as small as it can be.

For reading config.js we have to insert binary.min.js which is almost ~2MB and affects this page's loading speed.

We may change it, once we remove concatenation of js files.